projects/packages/jetpack-mu-wpcom/changelog/update-php-8.1

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+Janitorial change: added PHPCS setting file.
```

Comment:
I might change this to something like "Update minimum PHP version to 8.1." Same for the other change entry below. It doesn't matter a whole lot for these though.